Source/LibationCli/LibationCli.csproj

```diff
@@ -3,7 +3,8 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net7.0</TargetFramework>
+		<TargetFramework>net7.0-windows</TargetFramework>
```

Comment:
Interesting. How will `net7.0-windows` jive with linux?

Reply:
It doesn't matter for releases because they're built with the publish command and the release identifier is overridden via the command line argument. This is just for debugging on windows machines. It's needed for debugging so that both WindowsConfigApp and LibationCli can be built with the same output directory.